src/test/scala/nl/knaw/dans/easy/stage/fileitem/EasyStageFileItemSpec.scala

```diff
@@ -65,63 +69,13 @@ class EasyStageFileItemSpec extends FlatSpec with Matchers {
   }
 
   "main" should "report a configuration problem" in {
+    System.setProperty("app.home", "src/main/assembly/dist")
     val args = "src/test/resources/example.csv target/testSDO".split(" ")
     the[Exception] thrownBy EasyStageFileItem.main(args) should
       have message "no protocol: {{ easy_stage_dataset_fcrepo_service_url }}"
   }
 
-
-  it should "create expected file item SDOs in the multi-deposit use case (i.e. when file-location is provided)" in {
-    EasyStageFileItem.run(new FileItemSettings(
-      ownerId = Some("testOwner"),
-      sdoSetDir = Some(new File("target/testSDO")),
-      file = Some(new File("original/newSub/file.mpeg")), // TODO this may fail!
-      size = Some(1),
-      datasetId = Some("easy-dataset:1"),
-      pathInDataset = Some(new File("original/newSub/file.mpeg")),
-      format = Some("video/mpeg"),
-      subordinate = "object" -> s"info:fedora/easy-dataset:1",
-      easyFilesAndFolders = mockEasyFilesAndFolders(HashMap(
-        "easy-dataset:1 original/newSub/file.mpeg" -> Success("original", "easy-folder:1")
-      )),
-      fedora = mockFedora(HashMap(
-        "pid~easy-dataset:1" -> Seq("easy-dataset:1")
-      )),
-      accessibleTo = FileAccessRights.NONE,
-      visibleTo = FileAccessRights.ANONYMOUS
-    ))
-
-    // comparing with sample output
-
-    // TODO sdoDir "newSub" should have been "original_newSub" to avoid potential conflicts !!!
-    val actualSdoSet = Path("target/testSDO/easy-dataset_1")
-    val expectedSdoSet = Path("src/test/resources/expectedFileItemSDOsWithMultiDeposit")
-    getRelativeFiles(actualSdoSet) shouldBe getRelativeFiles(expectedSdoSet)
-    actualSdoSet.walk.toSeq.map(_.path).sortBy(s => s).zip(
-      expectedSdoSet.walk.toSeq.map(_.path).sortBy(s => s)
-    ).foreach {
-      case (actual, expected) if actual.endsWith("cfg.json") =>
-        readCfgJson(expected) shouldBe readCfgJson(actual)
-      case (actual, expected) if actual.endsWith("fo.xml") =>
-        readDatastreamFoxml(actual) shouldBe readDatastreamFoxml(expected)
-      case (actual, expected) => // metadata of a file or folder
-        readFlatXml(actual) shouldBe readFlatXml(expected)
-    }
-
-    // a less verbose check reworded
-
-    readDatastreamFoxml("target/testSDO/easy-dataset_1/newSub/fo.xml") shouldBe Set(
-      "dc_title" -> "original/newSub",
-      "prop_state" -> "Active",
-      "prop_label" -> "original/newSub",
-      "prop_ownerId" -> "testOwner")
-
-    // clean up
-
-    Path("target/testSDO").deleteRecursively()
-  }
-
-  it should "report a missing size" in {
+  "run" should "report a missing size" in {
     the[NoSuchElementException] thrownBy EasyStageFileItem.run(new FileItemSettings(
```

Comment:
I just deleted other tests, therefore I only changed `it` to a string for this one.